src/NServiceBus.AcceptanceTests/BestPractices/When_subscribing_to_command_bestpractices_disabled_on_endpoint.cs

```diff
@@ -7,6 +7,7 @@
 
     public class When_subscribing_to_command_bestpractices_disabled_on_endpoint : NServiceBusAcceptanceTest
     {
+        [Ignore("not supported via code first API")]
```

Comment:
I don't see the value in keeping these tests around if we're just going to ignore them. It seems like we should either make the code-first API work when best practices are disabled, or remove the tests altogether.